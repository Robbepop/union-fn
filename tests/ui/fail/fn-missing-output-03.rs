fn main() {}

#[union_fn::union_fn]
trait Foo {
    type Output = i32;

    fn foo() -> Self::Output {}
    fn bar() -> i32 {}
}
