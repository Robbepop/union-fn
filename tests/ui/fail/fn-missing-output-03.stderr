error: must return Self::Output
 --> tests/ui/fail/fn-missing-output-03.rs:8:17
  |
8 |     fn bar() -> i32 {}
  |                 ^^^

error: since Output is defined here
 --> tests/ui/fail/fn-missing-output-03.rs:5:19
  |
5 |     type Output = i32;
  |                   ^^^
